src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+
+  .logo {
+    height: 40px;
+    width: 40px;
+    object-fit: contain;
+    padding: 10px 0;
+  }
+
+  nav {
+    text-transform: uppercase;
+
+    .nav-list {
+      display: flex;
+      margin: 0;
+      padding: 0;
+      list-style: none;
```

Comment:
The `.nav-list` should have a `margin` and `padding` reset to `0` as per the code style guide. Good job on that. However, if there are no additional styles, the `margin: 0;` and `padding: 0;` declarations can be omitted, as they do not change the default browser styles for a `ul` element with `list-style: none;`.